js/desktopCapturer/getSource.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
```

Comment:
Is this just a new API that does the same thing as the `getSources` API? If yes, why reinvent it?

Reply:
@VikasShashidhar `getSources` returns an array of sources whereas `getSource` returns single source selected by the user. We are keeping the `getSources` API to support older versions of Electron and Client app.

We can remove `getSources` after 2-3 sprints 😄